sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -811,11 +811,16 @@ public sealed partial class DynamicData : System.Dynamic.IDynamicMetaObjectProvi
         System.Dynamic.DynamicMetaObject System.Dynamic.IDynamicMetaObjectProvider.GetMetaObject(System.Linq.Expressions.Expression parameter) { throw null; }
         public override string ToString() { throw null; }
     }
+    public enum DynamicDataDefaults
```

Comment:
This is an approach used by JsonSerializer, e.g. [JsonSerializerDefaults](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializerdefaults?view=net-8.0), and by [ASP.NET Core for web apps](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/configure-options?pivots=dotnet-7-0#web-defaults-for-jsonserializeroptions)

Reply:
Removed per discussion.